### PR TITLE
Added a section to the Glue job code to replace illegal characters fo…

### PR DESCRIPTION
**Pull request by sinanerdem**
…r column headers

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: labs/01_ingestion_with_glue/README.md [hunk 1]
```diff
@@ -29,11 +26,11 @@ All resources to be created **must** be in the same region.
 
 The encoding of your raw files should be UTF-8. You should export your files from your source with UTF-8 encoding. For this workshop, you may convert the encoding before uploading files to S3 bucket with text editing tools, such as Sublime Text
 
-or by using this Linux command: 
-``` python iconv -f <current-encoding of file> -t utf-8 data.csv outputfile.csv```
-    
+or by using this Linux command:
+`python iconv -f <current-encoding of file> -t utf-8 data.csv outputfile.csv`
+  
 if you don't know the encoding, you can use this command to determine: "
```
Comment (BastLeblanc) [inline labs/01_ingestion_with_glue/README.md]: extra " after determine

File: labs/01_ingestion_with_glue/README.md [hunk 2]
```diff
@@ -107,8 +104,7 @@ NOTE: “AWSGlueServiceRole” is an AWS Managed Policy to provide Glue with nee
 
 ### Creating a Development Endpoint and Notebook (First Part)
 
-
-In AWS Glue can create an environment—known as a development endpoint—that you can use to iteratively develop and test your extract, transform, and load (ETL) scripts. 
+In AWS Glue can create an environment—known as a development endpoint—that you can use to iteratively develop and test your extract, transform, and load (ETL) scripts.
```
Comment (BastLeblanc) [inline labs/01_ingestion_with_glue/README.md]: In AWS Glue**, you** can create an envi

```suggestion
In AWS Glue, you can create an environment—known as a development endpoint—that you can use to iteratively develop and test your extract, transform, and load (ETL) scripts.
```

**Review by BastLeblanc — COMMENTED** [review]
small typos, see comments inline

**Comment by sinanerdem** [comment]
Comments fixed with a second commit.

**Review by BastLeblanc — APPROVED** [review]
Approved

**Pull request merged**